### ISSUE TITLE
Use labels in this repo

### ISSUE THREAD

**Issue by friedger** [blockstack/app-mining#157]
The following labels could be used:
* meta (for issues about using github, managing requests, etc. This issue should have label meta)
* question
* new-reviewer
* dryrun
* app (for issues related to a single app)
* opinion (for issues describing subjective aspects of the app mining process)

**Comment by jeffdomke** [comment]
Good idea. We are trying to hire a new PM for the program and this person would likely own the labels and keeping things labeled. Suggest we wait on this until that person is here.
